app/src/main/java/ch/sdp/vibester/database/DataGetter.kt

```diff
@@ -207,6 +208,10 @@ class DataGetter @Inject constructor() {
         dbRoomRef.child(partyRoom.getRoomID()).child("emailList").setValue(partyRoom.getEmailList())
     }
 
+    fun getCurrentUser(): FirebaseUser? {
```

Comment:
Reminder to add documentation for functions that are not private, no matter how obvious they can be. Not too important as we need a general check over our entire codebase and write docs, but a small reminder